templates/collections.html

```diff
@@ -0,0 +1,32 @@
+{% set page_title = pages.collections.title %}
+{% set page_link = links.ansible_community %}
+{% extends "_base.html" %} {% block body %}
+<div class="container">
+  <div class="full-width-bg component">
+    <div class="grid-wrapper">
+      <div class="width-3-12 width-12-12-m">
+        <img class="page-logo" src="{{ base.base_url }}static/images/community_logo.svg">
+      </div>
+      <div class="width-9-12 width-12-12-m">
+        <div class="page-title">
+          <h2>{{ pages.collections.title }}</h2>
+          <p>{{ pages.collections.intro }}</p>
+        </div>
+      </div>
+      <div class="width-12-12 width-12-12-m">
+        <p>{{ pages.collections.redirect }}</p>
+        <ul class="fa-ul">
+          <li>
+            <span class="fa-li"><i class="fas fa-book"></i></span>
+            <a href="{{ index.quicklinks.collection_index.link }}" target="_blank">{{ index.quicklinks.collection_index.label }}</a>
```

Comment:
builtin modules are by far the most popular. Can we add a quick link here for https://docs.ansible.com/ansible/latest/collections/ansible/builtin/index.html#plugins-in-ansible-builtin